test/network/scenarios/common.js

```diff
@@ -17,6 +17,9 @@
 const childProcess = require('child_process');
 const utils = require('../utils');
 
+const NODE_READY_REGEX = /Finished sync/;
```

Comment:
If I only wait for 'Blockchain ready', it breaks the test.

Reply:
The name of regular expression you are using is `NODE_READY_REGEX ` https://github.com/LiskHQ/lisk/blob/ed88a37008d92b82260d86db956ec543c5c71eca/test/network/scenarios/common.js#L20

Which clearly is not the case, you are not waiting for the node to be ready, you are waiting for node to be synced at least once. If that's the case please use proper naming for the variables.

Reply:
>> Which clearly is not the case

It's not so black and white. 'Node ready' can mean a lot of things depending on whether we're looking at it from the perspective of the node or from the perspective of the test.

I'll rename to `NODE_FINISHED_SYNC_REGEX` to avoid any confusion.